style.css

```diff
@@ -0,0 +1,128 @@
+@media only screen and (max-width: 375px) {
+  .header {
+    width: 375px;
+    height: 51px;
+    background: #3c3a39;
+    display: flex;
+    position: absolute;
+    margin-left: -10px;
+    top: -1px;
+  }
+
+  .imagcss {
+    height: 15px;
+    width: 20px;
+    flex: none;
+    margin-left: 80px;
+  }
+
+  .toolbar {
+    margin-top: 18px;
+    flex-grow: 1;
+  }
+
+  .logo {
+    position: static;
+    width: 77px;
+    height: 15px;
+    left: 12px;
+    top: 10px;
+    margin-left: 20px;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 15px;
+    line-height: 15px;
+    color: #fff;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    padding: 60px 10px;
+    position: absolute;
+    width: 375px;
```

Comment:
- [ ] OPTIONAL: Please avoid using position absolute, because it takes the elements out of document flow and leads to bugs in further development. Here you can easily align everything with only flexbox properties.